TASVideos/Pages/Forum/Topics/_TopicPost.cshtml

```diff
@@ -54,21 +54,18 @@
 	<cardbody class="p-2">
 		<row>
 			<div class="forum-profile-section col-lg-2 col-md-3 col-4">
-				<div class="text-center">
+				<div class="text-center" style="margin-left: -8px;">
 					<h6>
 						<profile-link username="@Model.PosterName"></profile-link>
 					</h6>
 				</div>
 				<div class="card-text text-left" style="line-height: 1">
-					<small>
-						@string.Join(", ", Model.PosterRoles)
+					<small>@string.Join(", ", Model.PosterRoles.Select(s => s.Replace(' ', '\u00A0')))
```

Comment:
Is this so that something like "Site Admin" doesn't get text wrapped?

Reply:
yeah Yoshi suggested this so it doesn't wrap in the middle of a role